drivers/include/periph/gpio.h

```diff
@@ -169,22 +172,28 @@ int gpio_init(gpio_t pin, gpio_mode_t mode);
 int gpio_init_int(gpio_t pin, gpio_mode_t mode, gpio_flank_t flank,
                   gpio_cb_t cb, void *arg);
 
-#endif /* MODULE_PERIPH_GPIO_IRQ */
```

Comment:
Do this in a separate PR. The reason that I hid gpio_init_int only in #9845 was to let the user (and ci) to detect when the gpio IRQ module was used without being selected as not all gpio drivers are using conditional compilation.